src/logoutPage/logoutPage.component.tsx

```diff
@@ -77,48 +28,67 @@ interface LogoutPageDispatchProps {
   signIn: () => Action;
   signOut: () => void;
 }
-export type CombinedLogoutPageProps = LogoutPageProps &
-  LogoutPageDispatchProps &
-  WithStyles<typeof styles>;
 
-export const LogoutPageComponent = (
+export type CombinedLogoutPageProps = LogoutPageProps & LogoutPageDispatchProps;
+
+export const UnconnectedLogoutPage = (
   props: CombinedLogoutPageProps
 ): React.ReactElement => {
   const logout = (): void => {
     props.signOut();
   };
+
   return (
     <div className="logout-page">
-      <Paper className={props.classes.paper}>
+      <Paper
+        sx={(theme: Theme) => ({
+          marginTop: theme.spacing(8),
+          display: 'flex',
+          flexDirection: 'column',
+          alignItems: 'center',
+          padding: theme.spacing(3),
+          [theme.breakpoints.up(
+            400 + parseInt(theme.spacing(6).replace('px', ''))
+          )]: {
+            width: 400,
+            marginLeft: 'auto',
+            marginRight: 'auto',
+          },
+        })}
+      >
         {props.user.avatarUrl !== '' ? (
-          <Avatar
-            className={props.classes.avatarUrl}
-            alt="user"
-            src={props.user.avatarUrl}
-          />
+          <Avatar sx={{ margin: 1 }} alt="user" src={props.user.avatarUrl} />
         ) : (
-          <Avatar className={props.classes.avatar}>
+          <Avatar
+            sx={{
+              margin: 1,
+              backgroundColor: (theme: Theme) => theme.colours.lightBlue,
+              color: '#FFFFFF',
+            }}
+          >
             <AccountCircleIcon />
           </Avatar>
         )}
 
-        <Typography className={props.classes.textField}>
+        <Typography sx={{ marginTop: 1 }}>
           {getString(props.res, 'username-description')}
         </Typography>
-        <Typography className={props.classes.username}>
+        <Typography
+          sx={{ paddingTop: '3px', fontWeight: 'bold', fontSize: '17px' }}
+        >
           {props.user.username}
         </Typography>
-        <Typography className={props.classes.info}>
+        <InfoTypography>
```

Comment:
It's a bit odd to have a `styled` component that's only used once here, when the rest all use `sx` to style. Either they should all be `styled` or all done via `sx` I think